db/migrate/20160901233059_add_in_ord_status_cd_to_orders.rb

```diff
@@ -0,0 +1,13 @@
+class AddInOrdStatusCdToOrders < ActiveRecord::Migration
+  return if Order
```

Comment:
To be honest I'm not sure, I suppose it would work in either place.

I could try moving it, although it does work where it is.